spec/rspec_puppet_facts_spec.rb

```diff
@@ -377,31 +457,31 @@
 
         it { is_expected.to be_a(Hash) }
         it { is_expected.to have_attributes(:size => 1) }
-        it { is_expected.to include('windows-7-x64' => an_instance_of(Hash)) }
+        it { is_expected.to include('windows-7-x86_64' => an_instance_of(Hash)) }
       end
 
       context 'with a revision release' do
         let(:release) { ['2012 R2'] }
 
         it { is_expected.to be_a(Hash) }
         it { is_expected.to have_attributes(:size => 1) }
-        it { is_expected.to include('windows-2012 R2-x64' => an_instance_of(Hash)) }
+        it { is_expected.to include('windows-2012 R2-x86_64' => an_instance_of(Hash)) }
       end
 
       context 'with a Server prefixed release' do
         let(:release) { ['Server 2012'] }
 
         it { is_expected.to be_a(Hash) }
         it { is_expected.to have_attributes(:size => 1) }
-        it { is_expected.to include('windows-2012-x64' => an_instance_of(Hash)) }
+        it { is_expected.to include('windows-2012-x86_64' => an_instance_of(Hash)) }
```

Comment:
Curious why the platform change here

Reply:
It looks like the `hardwaremodel` value changed in the Facter 3.x series for Windows from `x64` to `x86_64`.